NightscoutServiceKit/NightscoutService.swift

```diff
@@ -21,12 +21,22 @@ public final class NightscoutService: Service {
 
     public weak var serviceDelegate: ServiceDelegate?
 
```

Comment:
This issue was discussed in https://github.com/LoopKit/Loop/issues/1536. 

But the gist is that a lazy getter poses issue on the first run that prevents the token from uploading. Since the Nightscout service is not setup the first time this is called, these values will be nil. That will prevent calls that use these values from working until you kill app and restart.